modules/ROOT/pages/ip-whitelist.adoc

```diff
@@ -7,68 +7,66 @@ endif::[]
 
-* Access based on the IP address of the request
-* Access based on the origin IP address of the x-Forwarded-For header
 
-== Whitelisting Based on the IP address of the Request
+[[specifying_ip_address]]
+== Specifying IP Addresses to Whitelist in Your Policy 
 
-. Click the instance number of an API.
-+
-. On the API dashboard, click Policies.
-+
-The list of any applied policies appears.
-+
-. In Apply New Policy.
-. In Select Policy, choose IP Whitelist, and click Configure Policy.
-. Specify an address or a range or addresses as follows:
-+
-* Define a specific IP address. For example, 192.168.1.1.
-* Define a subset of addresses by identifying a subnet mask, such as 192.168.3.1/30. This includes the consolidated range: 192.168.3.0 - 192.168.3.3.
-* Define a whole range of IP addresses by stating the relevant octets of the IP address you want to permit. For example, setting `192.168` will include IP addresses from 192.168.0.0 to 192.168.255.255.
+You can specify the IP addresses to approve in the policy using the following ways:
 
-To whitelist a list of IP addresses:
+* Whitelist access based on the IP address of the request
+* Whitelist access based on the origin IP address that is determined by resolving a specific DataWeave or MEL expression, such as the x-Forwarded-For header or the HTTP header.
+* Any other origin
 
-. Enter one or more IP addresses in *Whitelist*.
-+
-image::ip-whitelist-051a0.png[height=353,width=377]
-+
-. Click +.
-+
-The greyed-out IP address and a deletion control appears below the Whitelist text entry field.
-+
-//image::ip-whitelist-1b671.png[height=353,width=377]
-image::ip-whitelist-1b671.png[height=475,width=500]
+=== Whitelisting Access Based on the IP address of the Request
+
+To whitelist access based on the IP address of the request, you can:
+
+* Define a specific IP address by enumerating it in the white space, for example, `192.168.1.1`.
+* Define a subset of addresses by identifying a subnet mask. 
+For example, `192.168.3.1/30`, which includes the consolidated range: `192.168.3.0 - 192.168.3.3`.
+* Define a whole range of IP addresses by stating the relevant octets of the IP address you want to permit. 
 +
-. Click *Apply*.
+For example, setting `192.168` will include IP addresses from `192.168.0.0` to `192.168.255.255`.
+
+=== Whitelisting Access Based on the Origin IP Address of the x-Forwarded-For Header
+
+If the client connects to your API through an HTTP proxy or a load-balancer, you can blacklist the client's specific IP (the IP originating the request) instead of the address that appears in the request.
 
-== Whitelisting Based on the Origin IP Address of the x-Forwarded-For Header
+For example, suppose you want to whitelist `192.168.2.3` and the address of a client connecting through an HTTP proxy is `92.40.1.255`, the client requests appear with this public address using the proxy in such a case.
```

Comment:
For clarity sake, you may mention that `192.168.2.3` is the private IP while `92.40.1.255` is the public one.

Reply:
Done! Thanks!